sdcm/utils/common.py

```diff
@@ -242,14 +244,17 @@ def download_file(self, link, dst_dir):
 
 
 def get_latest_gemini_version():
+    """search latest version of gemini tool"""
     bucket_name = 'downloads.scylladb.com'
 
     results = S3Storage(bucket_name).search_by_path(path='gemini')
     versions = set()
     for result_file in results:
-        versions.add(result_file.split('/')[1])
+        versions.add(Version(result_file.split('/')[1]))
+
+    latest_version = sorted(versions)[-1]
```

Comment:
Did you thought about `max()`? Looks like it do be doing exactly what happen here.

Reply:
Thanks, fixed